SPEC.md

```diff
@@ -693,7 +693,7 @@ JSON Schema for the Image Manifest (app image manifest, ACI manifest), conformin
     * **os**, **arch** can together be considered to describe the syscall ABI this image requires. **arch** is meaningful only if **os** is provided. If one or both values are not provided, the image is assumed to be OS- and/or architecture-independent. Currently supported combinations are listed in the [`types.ValidOSArch`](schema/types/labels.go) variable, which can be updated by an implementation that supports other combinations. The combinations whitelisted by default are (in format `os/arch`): `linux/amd64`, `linux/i386`, `freebsd/amd64`, `freebsd/i386`, `freebsd/arm`, `darwin/x86_64`, `darwin/i386`. See the [Operating System spec](OS-SPEC.md) for the environment apps can expect to run in given a known **os** label.
 * **app** (object, optional) if present, defines the default parameters that can be used to execute this image as an application.
     * **exec** (list of strings, required) executable to launch and any flags (must be non-empty; the executable must be an absolute path within the app rootfs; ACE can append or override)
-    * **user**, **group** (string, required) indicates either the UID/GID or the username/group name the app should run as (freeform string). If the user or group field begins with a "/", the owner and group of the file found at that absolute path inside the rootfs is used as the UID/GID of the process.
+    * **user**, **group** (string, required) indicates either the UID/GID or the username/group name the app should run as (freeform string). If the user or group field begins with a "=" contains all numbers after the equal sign, then the number shall be treated as the numerical UID/GID to use for the process. If the user or group field begins with a "/", the owner and group of the file found at that absolute path inside the rootfs is used as the UID/GID of the process.  This conforms to POSIX standards as well as Windows standards, where usernames are not allowed to contain a slash or equal sign. If the string is a username or group, the uid/gid should be resolved using the containers own `/etc/passwd` or `/etc/group` only if the files exist. Example values for the fields include `root`, `=1000`, or `/usr/bin/ping`.
```

Comment:
> As an alternative, if you don't mind getting fancy, you can define this as a string-or-int. If string, look it up in passwd, if int, don't. I very much dislike ad hoc syntax.

you prefer ad hoc semantics? :-). I am also not thrilled about it, although it's a clever solution to supporting the different platform requirements. 

My alternative proposal would be for the spec to be more advisory/restrictive rather than explicitly supporting all these alternatives: i.e. these fields _must_ be alphanumeric (probably following the string-or-int logic), _should_ be platform-suitable (i.e. if you use a numeric on Windows, YMMV).

In the absence of more concrete use cases I am starting to question the utility of keeping this provision; it's convenient but arguably unnecessary:
> If the user or group field begins with a "/", the owner and group of the file found at that absolute path inside the rootfs is used as the UID/GID of the process.